### PR TITLE
Add simple Cilium Envoy Config CRD example

### PR DESCRIPTION
**Pull request by jrajahalme**
- Small fixes to INSTALLATION.md
- Add a Cilium Envoy Config CRD example with instructions

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -44,12 +44,14 @@ In order to use the `hubble` command to observe flows you will want to use port-
 cilium hubble port-forward & 
 ```
 
-[Install the Hubble CLI](https://docs.cilium.io/en/v1.11/gettingstarted/hubble_setup/#install-the-hubble-client), and then you can observe flows in the cluster with commands such as: 
+[Install the Hubble CLI](https://docs.cilium.io/en/v1.11/gettingstarted/hubble_setup/#install-the-hubble-client), and then you can continuously observe flows in the cluster with commands such as: 
 
 ```
-hubble observe --server localhost:4245
+hubble observe -f
```
Comment (lizrice) [inline INSTALLATION.md:50]: I think if you're using port-forward, you have to specify the server address (unless you're on the same machine in which case the default setting of the hubble socket works, but in that case I don't think the hubble port-forwarding would be necessary?)

File: ENVOY-LISTENER-EXAMPLE.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+- `envoy.filters.network.http_connection_manager`
+- `envoy.filters.network.mongo_proxy`
+- `envoy.filters.network.mysql_proxy`
+- `envoy.filters.network.tcp_proxy`
+- `envoy.stat_sinks.metrics_service`
+- `envoy.transport_sockets.raw_buffer`
+
+We will evolve the list of built in extensions based on customer
+feedback for those extensions to be available to be used from CEC
+CRDs.
```
Comment (lizrice) [inline ENVOY-LISTENER-EXAMPLE.md:149]: This is really useful - I'm thinking we should turn this into a Discussion for getting feedback

File: INSTALLATION.md (hunk 1 above)
Comment (jrajahalme) [reply INSTALLATION.md:50]: Port forward forwards a remote (or VM/container) port into a localhost port, and as long as the local port is the hubble default (4245) you do not need to specify the server when using port-forward.

**Comment by jrajahalme** [comment]
Renamed the `examples` directory as `custom-envoy-listener` and moved instructions to there as well (`README.md`). Also test drove the script in a local VM with Kind.

Marked as ready for review :-)

**Review by michi-covalent — APPROVED** [review]

File: INSTALLATION.md (hunk 1 above)
Comment (lizrice) [reply INSTALLATION.md:50]: Aha, the [default server changed ](https://github.com/cilium/hubble/commit/a1957ac11b09dd3bfd525ab3f8300df5623d3352)in 0.9.0 and I still had 0.8.0. You are correct!

**Pull request merged**